test-assets/test-projects/AntlrGeneratedFiles/Test.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Antlr4.CodeGenerator" Version="4.6.4" />
+    <PackageReference Include="Antlr4.Runtime" Version="4.6.4" />
+  </ItemGroup>
+  <ItemGroup>
+    <Antlr4 Include="Grammar.g4">
```

Comment:
:memo: This block will no longer be necessary with 4.6.5-beta002.

Reply:
FWIW, this is a test project directly from a customer. So, I want to ensure we don't regress it.